daemon/logger/jsonfilelog/read.go

```diff
@@ -81,9 +85,13 @@ func decodeFunc(rdr io.Reader) func() (*logger.Message, error) {
 			if err == io.ErrUnexpectedEOF {
 				reader := io.MultiReader(dec.Buffered(), rdr)
 				dec = json.NewDecoder(reader)
-				retries++
+				continue
```

Comment:
Why is retries not incremented? Both here and L73?

Reply:
These were erroneous. The for loop already increments retries.